src/JustEat.StatsD/StatsDPublisher.cs

```diff
@@ -23,7 +40,9 @@ public StatsDPublisher(StatsDConfiguration configuration)
             }
 
             _formatter = new StatsDMessageFormatter(configuration.Culture, configuration.Prefix);
-            _transport = new StatsDUdpTransport(configuration.Host, configuration.Port, null);
+
+            var endpointSource = EndpointParser.MakeEndPointSource(configuration.Host, configuration.Port, StatsDConfiguration.DefaultDnsLookupInterval);
```

Comment:
Seems a little odd to me here that we have an instance of ```StatsDConfiguration``` here, but we don't use it to get the endpoint lookup timeout and instead reference a static property. I get that it's not always relevant, it just _seems_ odd.